indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -352,6 +353,18 @@ def reviewing_state(self):
         else:
             return AbstractReviewingState.mixed
 
+    @property
+    def data_by_field(self):
+        return {value.contribution_field_id: value for value in self.field_values}
+
+    @property
+    def submitted_for_tracks_titles(self):
```

Comment:
we don't, iirc i already suggested using `join(.., attr=..)` in jinja in the places where the property is used atm